src/js/components/FrameworkConfiguration.js

```diff
@@ -319,6 +324,7 @@ class FrameworkConfiguration extends Component {
             {StringUtil.capitalize(packageDetails.getName()) +
               " " +
               packageDetails.getVersion()}
```

Comment:
I am not sure that this is what @danielone meant when he said "Next to service icon, have the date be part of the selected version number so "Version: 2.6.03.11. (2019-03-12)"
![Screenshot from 2019-08-23 14-34-40](https://user-images.githubusercontent.com/40791275/63589991-3e817480-c5b3-11e9-8c6f-bf81db80e164.png)